index.html

```diff
@@ -185,42 +211,44 @@ <h1 class="project-name">Base Apparel coming soon page</h1>
           <div class="fnl-btn">
             <button type="button" class="btn" disabled>See project</button>
           </div>
+          </div>
         </div>
       </section>
       <section class="visit">
-        <h2 class="cnct">CONTACT US</h2>
+        <h2 class="cnct1">CONTACT US</h2>
+        <h2 class="cnct2">Let's Chat</h2>
         <p class="note">If you have an application
           you are interested in developing
           , a feature that you need built
           or a project that needs coding.
-          I’d love to help with it
+          I'd love to help with it
         </p>
         <form action="https://formspree.io/f/mgedkakg" method="POST" class="prjct">
-          <input class="wrt" maxlength="30" type="text" placeholder="Enter your name" required>
-          <input class="wrt" type="Email" placeholder="Enter your mail" required>
+          <input class="wrt" maxlength="30" type="text" placeholder=" Enter your name" required>
+          <input class="wrt" type="Email" placeholder=" Enter your mail" required>
           <textarea class="textarea" maxlength="500"  name="comment"
-            placeholder="Enter your text"></textarea>
-          <button class="btn" type="submit">Submit</button>
+            placeholder=" Enter your text"></textarea>
+          <button class="btn frm " type="submit">Send</button>
         </form>
       </section>
       <section class="final">
```

Comment:
- [ ] I suggest you turn this tag into a `footer` tag for better semantics of the website, and also add some padding and increase the size of the social icons to better match the template.

| Current design  | Template design |
| ------------- | ------------- |
| ![](https://i.imgur.com/flnMzwL.png)  | ![](https://i.imgur.com/uitaiIN.png)  |